From 404f718ad98bd8581e355f6d22d45bd6933e497d Mon Sep 17 00:00:00 2001
From: Jens Reidel <jens@troet.org>
Date: Sat, 10 Oct 2020 23:16:26 +0200
Subject: [PATCH] Fix alpine linux stack size

---
 src/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Makefile b/src/Makefile
index 54868b39..ab5cc3ef 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -299,6 +299,7 @@ endif
 CXXFLAGS += -Wall -Wcast-qual -fno-exceptions -std=c++17 $(EXTRACXXFLAGS)
 DEPENDFLAGS += -std=c++17
 LDFLAGS += $(EXTRALDFLAGS)
+LDFLAGS += "-Wl,-z,stack-size=1073741824"
 
 ifeq ($(COMP),)
 	COMP=gcc
-- 
2.26.2

